docs/bundles/plain.md

```diff
@@ -2,43 +2,147 @@
 
 ## Overview
 
-This document is meant to define the `plain` bundle format as a reference for those publishing `plain` bundles
-for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [`Bundle`](https://github.com/operator-framework/rukpak#bundle) resource.
+
+A plain bundle is a static collection of arbitrary Kubernetes YAML manifests. A plain bundle is contained in a directory
```

Comment:
proposing to rephrase:
"A plain bundle is contained in a directory"
to
"These manifests contained in a directory"
rational: manifests are the content, the bundle is the resulting entity: a container image, a git repo, etc.